### ISSUE TITLE
Tooltip is incorrectly offset when its container has a CSS transform scale property

### ISSUE THREAD

**Issue by aost** [twbs/bootstrap#18226]
Here is an example: http://jsfiddle.net/uux31tay/2/

UPDATE: Fixed example according to @twbs-lmvtfy's comment.

**Comment by twbs-lmvtfy** [comment]
Hi @aost!

You appear to have posted a live example (http://fiddle.jshell.net/uux31tay/1/show/light/), which is always a good first step. However, according to [the HTML5 validator](http://html5.validator.nu), **your example has some validation errors**, which might potentially be causing your issue:
* JSFiddle inserts the contents of its "HTML" pane within ```` <body>...</body> ```` tags, so you cannot include your own ```` <body> ```` tag in your JSFiddle.
* line 40, column 3 thru column 8: Start tag ```` body ```` seen but an element of the same type was already open.
* line 48, column 1 thru column 7: Saw an end tag after ```` body ```` had been closed.

You'll need to **fix these errors** and post a revised example before we can proceed further.
Thanks!

(*Please note that this is a [fully automated](https://github.com/cvrebert/lmvtfy) comment.*)

**Comment by cvrebert** [comment]
Reposting in new comment to trigger revalidation: http://jsfiddle.net/uux31tay/2/

**Comment by twbs-lmvtfy** [comment]
Hi @cvrebert!

You appear to have posted a live example (http://fiddle.jshell.net/uux31tay/2/show/light/), which is always a good first step. However, according to [Bootlint](https://github.com/twbs/bootlint), **your example has some Bootstrap usage errors**, which might potentially be causing your issue:
* line 41, column 5: [W007](https://github.com/twbs/bootlint/wiki/W007): Found one or more `<button>`s missing a `type` attribute.
* line 42, column 5: [W007](https://github.com/twbs/bootlint/wiki/W007): Found one or more `<button>`s missing a `type` attribute.
* line 43, column 5: [W007](https://github.com/twbs/bootlint/wiki/W007): Found one or more `<button>`s missing a `type` attribute.
* line 44, column 5: [W007](https://github.com/twbs/bootlint/wiki/W007): Found one or more `<button>`s missing a `type` attribute.
* line 45, column 5: [W007](https://github.com/twbs/bootlint/wiki/W007): Found one or more `<button>`s missing a `type` attribute.
* line 13, column 3: [W013](https://github.com/twbs/bootlint/wiki/W013): Bootstrap version might be outdated. Latest version is at least 3.3.5 ; saw what appears to be usage of Bootstrap 3.2.0
* line 14, column 3: [W013](https://github.com/twbs/bootlint/wiki/W013): Bootstrap version might be outdated. Latest version is at least 3.3.5 ; saw what appears to be usage of Bootstrap 3.2.0

You'll need to **fix these errors** and post a revised example before we can proceed further.
Thanks!

(*Please note that this is a [fully automated](https://github.com/cvrebert/lmvtfy) comment.*)

**Comment by cvrebert** [comment]
Punting as WontFix per https://github.com/twbs/bootstrap/issues/15590#issuecomment-113321235
Thankfully, this bug doesn't apply to Bootstrap v4: http://jsfiddle.net/cvrebert/3md4zw4t/

**Comment by twbs-lmvtfy** [comment]
Hi @cvrebert!

You appear to have posted a live example (http://fiddle.jshell.net/cvrebert/3md4zw4t/show/light/), which is always a good first step. However, according to [Bootlint](https://github.com/twbs/bootlint), **your example has some Bootstrap usage errors**, which might potentially be causing your issue:
* line 63, column 5: [W007](https://github.com/twbs/bootlint/wiki/W007): Found one or more `<button>`s missing a `type` attribute.
* line 64, column 5: [W007](https://github.com/twbs/bootlint/wiki/W007): Found one or more `<button>`s missing a `type` attribute.
* line 65, column 5: [W007](https://github.com/twbs/bootlint/wiki/W007): Found one or more `<button>`s missing a `type` attribute.
* line 66, column 5: [W007](https://github.com/twbs/bootlint/wiki/W007): Found one or more `<button>`s missing a `type` attribute.
* line 67, column 5: [W007](https://github.com/twbs/bootlint/wiki/W007): Found one or more `<button>`s missing a `type` attribute.

You'll need to **fix these errors** and post a revised example before we can proceed further.
Thanks!

(*Please note that this is a [fully automated](https://github.com/cvrebert/lmvtfy) comment.*)
